cmd/snap/error.go

```diff
@@ -81,7 +81,13 @@ func fill(para string, indent int) string {
 	return strings.TrimSpace(buf.String())
 }
 
+const rebootingMsg = "snapd is in the process of rebooting the system"
```

Comment:
This can go away with the proposal. The default behavior of errorToCmdMessage will surface the error.Message field.